addon/components/tooltip-on-element.js

```diff
@@ -67,8 +67,11 @@ export default EmberTetherComponent.extend({
   'aria-hidden': computed.not('tooltipIsVisible'),
   attributeBindings: ['aria-hidden', 'role', 'tabindex'],
   classNameBindings: ['effectClass'],
-  classPrefix: 'tooltip',
-  classNames: ['tooltip'],
+  classPrefix: 'ember-tooltip',
+  classNames: ['ember-tooltip', 'tooltip'],
```

Comment:
I'm honestly fine switching straight from `.tooltip` to `.ember-tooltip` because there's no easy way enforce and warn about deprecations for classes. I don't have a problem issuing a breaking release for this (`2.0.0`).

Thus, I'd switch to classes like `.ember-tooltip-target-attached-left` and not support `.tooltip-target-attached-left`. I'd also remove `.tooltip` completely. However, if you want to support both I'm fine with that.